pyphi/tpm.py

```diff
@@ -137,7 +169,7 @@ def infer_cm(tpm):
     """Infer the connectivity matrix associated with a state-by-node TPM in
```

Comment:
This should be similarly generalized to work in both the nonbinary and binary cases.